### PR TITLE
Bug fix: Buttons colors not changing

### PR DESCRIPTION
**Pull request by Markytt**
### Description
Home button, Spawn button, and Back button wouldn't change colors when color changes are made in color manager

### Objectives
- [x] Make button colors change 
- [x] Make color of button text and score board text change 

### Testing Done
- Changed Background HUD and Main Text multiple times to make sure the buttons are changing to the correct colors 

[JIRA Issue](https://jira.autodesk.com/browse/AARD-XXXX)

### CODE REVIEW

File: engine/Assets/Scripts/UI/Dynamic/Panels/Information/ScoreboardPanel.cs [hunk 1]
```diff
@@ -18,8 +19,8 @@ public class ScoreboardPanel : PanelDynamic {
 
     private readonly bool _showTimer;
 
-    private Label time, redScore, blueScore;
-    private Content topContent, bottomContent;
+    private static Label _time, _redScore, _blueScore, _redLabel, _blueLabel;
```
Comment (LucaHaverty) [inline engine/Assets/Scripts/UI/Dynamic/Panels/Information/ScoreboardPanel.cs:22]: Should these be static? If AssignColors is made not static, then I don't think these would have to be

File: engine/Assets/Scripts/UI/Dynamic/Panels/Information/ScoreboardPanel.cs [hunk 2]
```diff
@@ -114,13 +117,22 @@ public ScoreboardPanel(bool showTimer = true)
...
+        _redScore.SetText(Scoring.redScore.ToString());
+        _blueScore.SetText(Scoring.blueScore.ToString());
     }
 
-    public override void Delete() {}
+    public override void Delete() {
+        EventBus.RemoveTypeListener<ColorManager.OnThemeChanged>(AssignColors);
+    }
+
+     public static void AssignColors(IEvent e) {
```
Comment (LucaHaverty) [inline engine/Assets/Scripts/UI/Dynamic/Panels/Information/ScoreboardPanel.cs:132]: This too?

File: engine/Assets/Scripts/UI/Dynamic/Panels/Information/ScoreboardPanel.cs (hunk 1 above)
Comment (Markytt) [reply engine/Assets/Scripts/UI/Dynamic/Panels/Information/ScoreboardPanel.cs:22]: Maybe AssignColors could be made not static, but I don't think I'm knowledgeable enough to confidently make that change, so for now, the variables need to be static so there won't be errors.

**Review by a-crowell — APPROVED** [review]
Looks good, great work Mark!

**Review by BrandonPacewic — APPROVED** [review]
Excellent!

**Pull request merged**